.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build Image
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./Dockerfile
+          builder: ${{ steps.buildx.outputs.name }}
+          # Do not publish the image.
+          push: false
+          # Set the tag.
```

Comment:
Comment could be more descriptive. I'm new to this so maybe its obvious, but why is setting the tag important?

Reply:
I agree. The comment isn't so clear. We need the tag to recover the built docker image in later steps. As we don't have the deploy step in this PR (only in #5), I'm removed the tag from here and I'll add a better comment there.